tools/flash_shelly.py

```diff
@@ -0,0 +1,499 @@
+import getopt, importlib, platform, urllib, json, logging, re
+from sys import argv, exit
+from os import path, remove
+from subprocess import Popen, PIPE
+from time import sleep
+from importlib import util
+
+"""
```

Comment:
no need to make it a string. put it above imports and put `#!/usr/bin/env python3` as the first line

Reply:
that stop windows from running script.

Reply:
https://github.com/mongoose-os-apps/shelly-homekit/pull/176/commits/215e8e65fb1b07da10257e558cfad61f55ace32e

Reply:
ok after reading up, it seems it won't effect windows at all, so I'll add it, thx

Reply:
https://github.com/mongoose-os-apps/shelly-homekit/pull/176/commits/ae906910935bd41b57675857f7d42251938dad4c